front/src/config/constants.json

```diff
@@ -1,3 +1,7 @@
 {
-  "emailRegex": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$"
+  "emailRegex": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$",
+  "serverImagesPath": {
+    "avatars": "avatars"
+  },
+  "MAX_AVATAR_SIZE_MB": 2
```

Comment:
I think that const file should be in js file with named exports, not in json format

Reply:
I've added fileSettings.js and moved regular expression validation to validation.js.

Reply:
so, it is better to remove this file and remove using of this json

Reply:
removed